functional/measured-boot-swtpm-sanity/test.sh

```diff
@@ -85,7 +85,7 @@ rlJournalStart
         #rlRun "tsseventextend -tpm -if /var/tmp/binary_bios_measurements"
         rlRun -s "keylime_tenant -u $AGENT_ID --verify --tpm_policy '{}' --runtime-policy policy.json -f /etc/hostname -c add --mb_refstate mb_refstate2.txt"
         rlRun "limeWaitForAgentStatus $AGENT_ID 'Tenant Quote Failed'"
-        rlAssertGrep "keylime.tpm - ERROR - For PCR 0 and hash sha256 the boot event log has value '.*' but the agent returned '.*'" $(limeVerifierLogfile) -E
+        rlAssertGrep "keylime.tpm - ERROR - For PCR 0 and hash sha256 the boot event log has value '.*' but the agent .* returned" $(limeVerifierLogfile) -E
```

Comment:
With spaces on both sides of `.*` this assert won't match the old message format. You should remove one of them.
Also, please but back the pattern after "returned" if this part is still there. This change is not about different wording but about additional details about the agent added to the log message.